extensions/ql-vscode/package.json

```diff
@@ -398,22 +398,22 @@
         {
           "command": "codeQLDatabases.chooseDatabaseFolder",
           "when": "view == codeQLDatabases",
-          "group": "navigation"
+          "group": "1_open@1"
```

Comment:
Is it possible to create your own group below an icon, rather than the overflow menu? I think decluttering the database view is a good idea, but I am concerned users will not know to look in the overflow menu for 'Add Database' commands, whereas an 'Add Database From...' submenu may be more obvious.

Reply:
I don't know if that's possible. Let me look.

Reply:
No. It's not. https://code.visualstudio.com/api/extension-guides/tree-view#view-actions

I could put the actions in both places, but that seems worse. I feel like I would want to talk to a UX engineer for this.